readme.md

```diff
@@ -1,4 +1,4 @@
-# CheatSheet
+\_\_# CheatSheet
```

Comment:
Hmm this looks like an error

Reply:
yeah, that's odd... I'll revert this one and remove `*.md` from the list to format on commit

Reply:
I gotchu